From c2af3d7d970289f120279dad6b54604a263dd52c Mon Sep 17 00:00:00 2001
From: Zhichao Yu <zhichao.yu@rock-chips.com>
Date: Thu, 7 May 2020 17:07:03 +0800
Subject: [PATCH] ARM: dts: rockchip: modify romfs decompress memory for
 rv1126-evb-ddr3-v10-tb

We need more memory for romfs decompress. Currently, with NPU the romfs
size is 18MB.

Signed-off-by: Zhichao Yu <zhichao.yu@rock-chips.com>
Change-Id: Id473db768fc1994c606b3de9320f939da7123d11
---
 arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
index 870f9367bad2..c835016ae571 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
@@ -44,11 +44,11 @@
 		};
 
 		ramdisk_r: ramdisk@2800000 {
-			reg = <0x02800000 (32 * 0x00100000)>;
+			reg = <0x02800000 (40 * 0x00100000)>;
 		};
 
 		ramdisk_c: ramdisk@5800000 {
-			reg = <0x05800000 (16 * 0x00100000)>;
+			reg = <0x05800000 (20 * 0x00100000)>;
 		};
 	};
 };
-- 
2.35.3

